portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyTestFile.path

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>MyTestFile</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyTestFile</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>BUTTON_MENU</td>
+	<td>//button[contains(@data-qa-id, 'applicationsMenu')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BUTTON_CONTROL_PANEL</td>
+	<td>//*[text()='Control Panel']</td>
```

Comment:
I really like that you have used the text() to control the naming, I would just add the father attribute to keep padronized: 
**//button**//*[text()='Control Panel']